src/player_display.cpp

```diff
@@ -532,18 +536,22 @@ static void draw_stats_info( const catacurses::window &w_info, const Character &
         fold_and_print( w_info, point( 1, 1 + lines ), FULL_SCREEN_WIDTH - 2, c_light_gray,
                         display::weight_long_description( you ) );
     } else if( line == 5 ) {
+        // NOLINTNEXTLINE(cata-use-named-point-constants)
+        const int lines = fold_and_print( w_info, point( 1, 0 ), FULL_SCREEN_WIDTH - 2, c_magenta,
+                                          _( "How healthy you feel.  Excercise, vitamins, sleep and not ingesting poison will increase this overtime." ) );
```

Comment:
Are we missing a `fold_and_print` or similar here? I can't see `lines` being used anywhere. (Neither could my compiler)